app/assets/stylesheets/application.sass

```diff
@@ -19,9 +19,11 @@
 @import code
 @import faq
 @import footer
+@import graphs
```

Comment:
Any reason to not move all of the graphs into this folder now?

Reply:
Nope just laziness, thanks for calling me out on it. Moving!